requirements/requirements.txt

```diff
@@ -3,3 +3,5 @@ torchvision>=0.4.0
 tqdm
 psutil
 tensorboardX==1.8
+cupy-cuda100
+mpi4py
```

Comment:
Let's move these into a specific requirements.txt for 1bit adam. I'll take care of this though.